platform/src/legacy/legacy_platform_proxifier.ts

```diff
@@ -92,13 +92,13 @@ export class LegacyPlatformProxifier extends EventEmitter {
   /**
    * Neither new nor legacy platform should use this method directly.
    */
-  getConnections(callback: (error?: Error, count?: number) => void) {
+  getConnections(callback: (error: Error | null, count?: number) => void) {
     // This method is used by `even-better` (before we start platform).
     // It seems that the latest version of parent `good` doesn't use this anymore.
     if (this.server) {
       this.server.getConnections(callback);
```

Comment:
`getConnections` now requires `error | null`, not `error | undefined`, see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3c5b3e99085c7306532956cd7a33836649e743f6/types/node/v8/index.d.ts#L2731